src/index.html

```diff
@@ -22,6 +22,390 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <a
+        class="logo-link"
+        href="#"
+      >
+        <img
+          alt="Company logo"
+          class="logo_image"
+          src="./images/logo.png"
```

Comment:
The alt attribute 'Company logo' is appropriate here, but ensure that the image path './images/logo.png' is correct and the image exists at that location.